src/quemb/kbe/lo_k.py

```diff
@@ -163,13 +185,22 @@ def get_pao_k(Ciao, S, S12):
 
 def get_pao_native_k(Ciao, S, mol, valence_basis, ortho=True):
     """
-    Args:
-        Ciao: output of :func:`get_iao_native`
-        S: ao ovlp matrix
-        mol: mol object
-        valence basis: basis used for valence orbitals
-    Return:
-        Cpao (symmetrically orthogonalized)
+
+    Parameters
+    ----------
+    Ciao :
+        output of :code:`get_iao_native`
```

Comment:
@lweisburn to which `get_iao_native` should this refer?

Reply:
`get_iao_native` is an option in frankenstein, but not quemb. The `get_iao_native` I _think_ should be the same as `get_iao` in `molbe/lo`, but I am not sure and need to actually test that

Reply:
In this case I would just leave it as 
```
:code:`get_iao_native`
```
and merge. This way it is rendered as code, but does not link to any function in the code.
That is not an ideal situation, but it was the way the code was before and I think it makes no sense to postpone this PR for it.

Are you ok with that?

Reply:
I am okay with this for now! We will have to go through the IAO code more thoroughly later on